com.gzoltar.cli/pom.xml

```diff
@@ -67,6 +67,12 @@
       <artifactId>testng</artifactId>
       <version>${testng.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.github.classgraph</groupId>
+      <artifactId>classgraph</artifactId>
+      <version>4.8.28</version>
```

Comment:
Any reason to use 4.8.28 instead of the latest version available, i.e., 4.8.104?

Reply:
Thank you. I just tried with version 4.8.104 and it works good also.